.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust: [stable, beta, nightly, 1.73.0]
+        rust: [stable, beta, nightly, 1.74.0]
```

Comment:
rust-script's msrv is not 1.74:
```
error: failed to compile `rust-script v0.35.0`, intermediate artifacts can be found at `/tmp/cargo-install75pEMP`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.

Caused by:
  package `clap_lex v0.7.2` cannot be built because it requires rustc 1.74 or newer, while the currently active rustc version is 1.73.0
```